templates/admin/authorize.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Site search authorize admin template.
+ *
+ * @var \Swiftype\SiteSearch\Wordpress\Admin\Page $this
+ */
+?>
+
+<div class="wrap">
+
+    <?php include('common/header.php'); ?>
+
+    <div class="swiftype-admin">
+        <div class="main-content">
+            <h3><?= __('Thanks for installing the Site Search Plugin for Wordpress !'); ?></h3>
+            <div class="card">
+                <p><?= __("Please enter your Site Search API key in the field below and click 'Authorize' to get started."); ?></p>
```

Comment:
I think we should strip white space from the beginning and end of the key that is entered, because they'll be copy / pasting. If they have an extra space at the beginning they'll receive an "invalid key" error and it probably won't be clear as to why. This happened to me.

Reply:
Done in dc86928